src/s1ap/handlers/erab_setup_response.c

```diff
@@ -0,0 +1,136 @@
+            {
+                log_msg(LOG_INFO,
+                    "ERAB Setup Response S1AP_IE_MME_UE_ID %d\n", 
+                erab_su_resp_ies.data[i].val.mme_ue_s1ap_id);
+
+                erab_su_resp.ue_idx = erab_su_resp_ies.data[i].val.mme_ue_s1ap_id;
+            }
+            break;
+            case S1AP_IE_ENB_UE_ID:
+            {
+                log_msg(LOG_INFO,
+                    "ERAB Setup Response S1AP_IE_ENB_UE_ID %d\n",
+                erab_su_resp_ies.data[i].val.enb_ue_s1ap_id);
+
+                erab_su_resp.s1ap_enb_ue_id = erab_su_resp_ies.data[i].val.enb_ue_s1ap_id;
+            }
+            break;
+            case S1AP_IE_E_RAB_SETUP_LIST_BEARER_SU_RES:
+            {   
+                log_msg(LOG_INFO,
+                    "ERAB Setup Response S1AP_IE_E_RAB_SETUP_LIST_BEARER_SU_RES \
+		             received with the count : %d\n", 
+                     erab_su_resp_ies.data[i].val.erab_su_list.count);
+
+                erab_su_resp.msg_data.erabSuResp_Q_msg_m.erab_su_list.count =
+                    erab_su_resp_ies.data[i].val.erab_su_list.count;
+                memcpy(
+                    &erab_su_resp.msg_data.erabSuResp_Q_msg_m.erab_su_list.erab_su_item,
+                    &erab_su_resp_ies.data[i].val.erab_su_list.erab_su_item,
+                    sizeof(erab_su_resp_ies.data[i].val.erab_su_list.erab_su_item));
+            }
+            break;
+            case S1AP_IE_E_RAB_FAILED_TO_SETUP_LIST_BEARER_SU_RES:
+            {
+                log_msg(LOG_INFO,
+                    "ERAB Setup Response S1AP_IE_E_RAB_FAILED_TO_SETUP_LIST_BEARER_SU_RES \
+		             received with the count : %d\n", 
+                     erab_su_resp_ies.data[i].val.erab_fail_list.count);
+
+                erab_su_resp.msg_data.erabSuResp_Q_msg_m.erab_fail_list.count =
+                    erab_su_resp_ies.data[i].val.erab_fail_list.count;
+                memcpy(
+                    &erab_su_resp.msg_data.erabSuResp_Q_msg_m.erab_fail_list.erab_fail_item,
+                    &erab_su_resp_ies.data[i].val.erab_fail_list.erab_fail_item,
+                    sizeof(erab_su_resp_ies.data[i].val.erab_fail_list.erab_fail_item));
+            }
+            break;
+            default:
+                log_msg(LOG_WARNING, "Unhandled IE %d\n", erab_su_resp_ies.data[i].IE_type);
+        }
+    }
+
+    erab_su_resp.msg_type = erab_setup_response;
+    erab_su_resp.destInstAddr = htonl(mmeAppInstanceNum_c);
+    erab_su_resp.srcInstAddr = htonl(s1apAppInstanceNum_c);
+
+    //dump_erab_su_resp(&erab_su_resp.msg_data.erabSuResp_Q_msg_m);
+    int i = send_tipc_message(ipc_S1ap_Hndl, mmeAppInstanceNum_c,
+            (char*) &erab_su_resp,
+            S1_READ_MSG_BUF_SIZE);
+    if (i < 0)
+    {
+        log_msg(LOG_ERROR, "Error To write in erab_su_resp_handler %s\n", strerror(errno));
+    }
+
+    log_msg(LOG_INFO, "ERAB Setup Response sent to mme-app. Bytes sent %d\n", i);
```

Comment:
```suggestion else {
    log_msg(LOG_INFO, "ERAB Setup Response sent to mme-app. Bytes sent %d\n", i);
    } 
```

Reply:
Could you please clarify the difference between the suggested and the existing change?